node-playwright-chrome/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:14-slim
+ARG NODE_VERSION=14
+# Use buster to be consistent across node versions.
```

Comment:
What does this mean? That `slim` does not work for all versions?

Reply:
this means something different than the next comment. In this case, the buster is explicitly specified because `node:14-slim` is by default build on Debian stretch and `node:16-slim` on buster.

Reply:
🤯